compiler/src/iree/compiler/InputConversion/StableHLO/StableHLOToLinalgRandom.cpp

```diff
@@ -0,0 +1,591 @@
+// Copyright 2023 The IREE Authors
+//
+// Licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+// Implements logic for lowering StableHLO random number generation to Linalg
+// dialect.
+
+#include "iree/compiler/InputConversion/StableHLO/LegalizeToLinalgUtils.h"
+#include "iree/compiler/InputConversion/StableHLO/Rewriters.h"
+#include "mlir/Dialect/Linalg/IR/Linalg.h"
+#include "mlir/IR/BuiltinTypes.h"
+#include "mlir/Support/LogicalResult.h"
+#include "mlir/Transforms/DialectConversion.h"
+#include "stablehlo/dialect/StablehloOps.h"
+
+namespace mlir::iree_compiler::stablehlo {
+namespace {
+namespace stablehlo = mlir::stablehlo;
```

Comment:
Do we need the `namespace stablehlo = mlir::stablehlo;`? I haven't seen that pattern elsewhere in IREE code. Looks like it doesn't save _that_ much typing here.

Reply:
Does it make the confusing or ambiguous? Given that it's in an anonymous namespace and does not leak, I like that it makes repetitive code shorter (here and in the other stablehlo lowering files where it's also used).

Reply:
It makes it confusing to me, but I can overlook it...

https://abseil.io/tips/119
https://abseil.io/tips/130
https://google.github.io/styleguide/cppguide.html#Namespace_Names

Seems like this could be
```c++
namespace mlir {
namespace iree_compiler {
namespace stablehlo {
```

or 
```c++
namespace mlir {
namespace stablehlo {
```

(if we control all the code, we can pick whatever nesting we want, if some code still lives elsewhere then we're stuck with what namespace it is already using)